docs/content/en/docs/getting-started.md

```diff
@@ -0,0 +1,463 @@
+---
+title: Getting started with KLT
```

Comment:
```suggestion
title: Getting started with KLT Observability
```

Reply:
Small detail but this page gets users started with KLT (wholesale). Observability is one piece of KLT so I'm not going to change the titl.